packages/react/src/components/builder-component.component.tsx

```diff
@@ -126,37 +126,149 @@ const Throttle = decorator(throttle);
 const fetchCache: { [key: string]: any } = {};
 
 export interface BuilderComponentProps {
+  /**
+   * @package
+   * @deprecated use {@link name} instead.
```

Comment:
is `{@link name}` supposed to be `{@link model}` instead, since `name` and `modelName` are deprecated?

Reply:
d'oh, yea you're right.